### DISCUSSION TITLE
milvus集群中协调器的组件能怎样扩容呢

### DISCUSSION THREAD

**Discussion by xkz130** [milvus-io/milvus#22256] [Q&A and General discussion]
部署milvus集群发现只能扩容node相关的服务组件，而协调器相关的组件时无法扩容，协调器的组件能怎样扩容呢

**Comment by xiaofan-luan** [comment]
> 部署milvus集群发现只能扩容node相关的服务组件，而协调器相关的组件时无法扩容，协调器的组件能怎样扩容呢

可以纵向扩容，因为coordinator本身并不放数据，也不做查询，因此不需要太大。正常1c2G或者2c4G就可以

**Reply by xkz130** [reply]
不支持高可用，只部署一个没意义

**Reply by xiaofan-luan** [reply]
Milvus的高可用是基于K8s做的，2.2.3也支持了主备模式。开业一期多个

**Reply by xkz130** [reply]
请问有主备模式的相关部署文档链接么

**Reply by xkz130** [reply]
milvus的docker模式部署是否能多集群部署呢

**Reply by xiaofan-luan** [reply]
docker 模式不能部署集群版，必须走K8s

**Reply by xkz130** [reply]
为啥必须走K8S，是docker没有进行支持么

**Reply by xiaofan-luan** [reply]
目前暂时没有，所有扩缩容都是K8s支持，你可以使用使用多个docker然后走ansible部署

**Reply by xkz130** [reply]
收到
